src/components/Onboarding/Screens/GardenMetadata.js

```diff
@@ -1,22 +1,593 @@
-import React from 'react'
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react'
+import {
+  Box,
+  Button,
+  Field,
+  GU,
+  Help,
+  IconPlus,
+  IconTrash,
+  Info,
+  TextInput,
+  textStyle,
+  useTheme,
+} from '@1hive/1hive-ui'
 import { useOnboardingState } from '@providers/Onboarding'
+import { Header } from '../kit'
+import ImageUploader from '../../ImageUploader'
 import Navigation from '../Navigation'
+import { publishNewDao } from '../../../services/github'
 
-function GardenMetadata({ title }) {
-  const { onBack, onNext, step, steps } = useOnboardingState()
+const COMMUNITY_LINK_TYPE = 'community'
+const DOCUMENTATION_LINK_TYPE = 'documentation'
+
+const GARDEN_LOGO_TYPE = 'logo_type'
+const GARDEN_LOGO = 'logo'
+const TOKEN_LOGO = 'token_logo'
+
+const URL_REGEX = new RegExp(
+  /https?:\/\/(www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)/
+)
+
+function GardenMetadata() {
```

Comment:
Note we are not indicating to the user that we have store the image data when we go back and forth between screens.

What you think of having a `useEffect` that evaluates the config data we have stored on each type of asset. If it's not the default it means there were assets previously store and we can safely show a green check.

We can evaluate the `setFormData` and check the fields `[type]: base64` for every asset, comparing with the empty value.